openshift.Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Build the manager binary
+FROM golang:1.16 as builder
+ARG VERSION
+
+WORKDIR /workspace
+# Copy the Go Modules manifests
+COPY go.mod go.mod
+COPY go.sum go.sum
+# cache deps before building and copying source so that we don't need to re-download as much
+# and so that source changes don't invalidate our downloaded layer
+RUN go mod download
+
+# Copy the go source
+COPY main.go main.go
+COPY api/ api/
+COPY controllers/ controllers/
+
+# Build
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -ldflags "-s -w -X main.version=${VERSION}" -a -o manager main.go
+
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+WORKDIR /
+COPY --from=builder --chown=65532:65532 /workspace/manager .
```

Comment:
I think we need to remove `chown`? `65532` is not in UBI.

Reply:
Good catch, thanks Luca